src/main/java/pl/sknikod/kodemy/category/Category.java

```diff
@@ -0,0 +1,64 @@
+package pl.sknikod.kodemy.category;
+import pl.sknikod.kodemy.section.Section;
+
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.Set;
+import java.util.HashSet;
+
+@Entity
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
+    private long id;
+    private String name;
+    @OneToMany(fetch = FetchType.EAGER, cascade = {
```

Comment:
Na razie proponuję zostawić podstawową adnotację @OneToMany bez szczegółów. Powodem jest to iż sposoby ładowania encji i wymuszania kaskadowości, pasowałoby jeszcze przegadać po połączeniu zadań.